ext/hal/nordic/nrfx_glue.h

```diff
@@ -233,16 +233,39 @@ extern "C" {
 //------------------------------------------------------------------------------
 
 /** @brief Bitmask that defines DPPI channels that are reserved for use outside of the nrfx library. */
-#define NRFX_DPPI_CHANNELS_USED  0
+#define NRFX_DPPI_CHANNELS_USED 0
```

Comment:
@anangl I would like to see if these macros could be conditionally defined, only for 91.